security-oauth2/src/main/java/io/micronaut/security/oauth2/endpoint/token/response/TokenResponseExpiration.java

```diff
@@ -43,28 +40,12 @@ public TokenResponseExpiration(TokenResponse tokenResponse) {
         setTokenType(tokenResponse.getTokenType());
     }
 
-    /**
-     * Populates Expiration date with the expires in value.
-     *
-     * @param expiresIn The lifetime in seconds of the access token.
-     */
-    @Override
-    public void setExpiresIn(Integer expiresIn) {
```

Comment:
This is a breaking change as far as I can tell.

Reply:
I don't think it is a breaking change because there is an implementation of `setExpiresIn` in `TokenResponse` the parent class of this class. Thus, `TokenResponseExpiration` still exposes a `setExpiresIn` method.